chirp/drivers/uv5r.py

```diff
@@ -2025,12 +2025,16 @@ class TDUV5RRadio(chirp_common.Alias):
     MODEL = "TD-UV5R TriPower"
 
 
+class UV5R8WTri(chirp_common.Alias):
+    VENDOR = "Baofeng"
+    MODEL = "UV-5R 8W TriPower"
+
 @directory.register
 class BaofengBFF8HPRadio(BaofengUV5R):
     VENDOR = "Baofeng"
     MODEL = "BF-F8HP"
     ALIASES = [RT5_TPAlias, ROGA5SAlias, UV5XPAlias, TSTIF8Alias,
-               TenwayUV5RPro, TSTST9Alias, TDUV5RRadio]
+               TenwayUV5RPro, TSTST9Alias, TDUV5RRadio, UV5R8WTri]
```

Comment:
This is the old way to do in-app aliases, which is not how they should be done anymore (I need to update the developer docs probably).

However, if this is just a straight "choose X instead of Y" as it seems from your posts on the matter, please do this in `chirp/share/model_alias_map.yaml` which affects the website list only.